services/user.py

```diff
@@ -0,0 +1,60 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> User:
+    user = get_user_model().objects.create_user(
+        username=username,
+        password=password
+    )
+    if email:
+        user.email = email
+
+    if first_name:
+        user.first_name = first_name
+
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(id=user_id)
```

Comment:
Same here, you need to use get_user_model()
```suggestion
    return get_user_model().objects.get(pk=user_id)
```